uint64 timestamp
string agent_name
uint32 detour_steps
uint32 arrival_step
uint64 related_id
